tests/qnn/conftest.py

```diff
@@ -44,9 +44,9 @@ def circuit(inputs, w1, w2, w3, w4, w5, w6, w7):
         qml.templates.StronglyEntanglingLayers(w1, wires=list(range(n_qubits)))
         qml.RX(w2[0], wires=0 % n_qubits)
         qml.RX(w3, wires=1 % n_qubits)
-        qml.Rot(*w4, wires=2 % n_qubits)
+        qml.Rot(w4[0], w4[1], w4[2], wires=2 % n_qubits)
```

Comment:
Is it not supported anymore?

Reply:
It's because Tensorflow doesn't allow iteration over abstract tensors. The old code only works when the `KerasLayer` is not wrapped with `tf.function`, but the new code works for both.